instrumentation/koala/Gemfile

```diff
@@ -13,6 +13,7 @@ gem 'opentelemetry-instrumentation-base', path: '../base'
 
 group :test do
   gem 'byebug'
+  gem 'faraday', '<2.0'
```

Comment:
Would this make more sense to be included in the appraisals file instead? 

That would be true for cases where older versions of faraday would be compatible only with older versions of the koala gem no?

Reply:
hm, yea that probably makes more sense. practically speaking i think that the outcome of this issue comment https://github.com/arsduo/koala/issues/659#issuecomment-1005192539 will probably resolve this for us in general, but can move into appraisal for now.